src/main/java/com/example/W2Assessment/entity/Pet.java

```diff
@@ -0,0 +1,99 @@
+package com.example.W2Assessment.entity;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+@Entity
+@Table(name = "pet")
+public class Pet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "breed")
+    private String breed;
+
+    @Column(name = "date_created")
+    private LocalDate dateCreated;
+
+    @Column(name = "date_modified")
+    private LocalDate dateModified;
+
+    @ManyToOne(cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.DETACH})
```

Comment:
(cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.DETACH}
this is not needed here